lib/scala/kg/src/main/scala/io/github/tetherlessworld/mcsapps/lib/kg/stores/postgres/PostgresKgCommandStore.scala

```diff
@@ -0,0 +1,81 @@
+package io.github.tetherlessworld.mcsapps.lib.kg.stores.postgres
+
+import com.google.inject.Inject
+import io.github.tetherlessworld.mcsapps.lib.kg.formats.kgtk.KgtkEdgeWithNodes
+import io.github.tetherlessworld.mcsapps.lib.kg.models.edge.KgEdge
+import io.github.tetherlessworld.mcsapps.lib.kg.models.node.KgNode
+import io.github.tetherlessworld.mcsapps.lib.kg.models.path.KgPath
+import io.github.tetherlessworld.mcsapps.lib.kg.models.source.KgSource
+import io.github.tetherlessworld.mcsapps.lib.kg.stores.{KgCommandStore, KgCommandStoreTransaction}
+import play.api.db.slick.DatabaseConfigProvider
+
+import scala.concurrent.ExecutionContext
+
+class PostgresKgCommandStore @Inject()(dbConfigProvider: DatabaseConfigProvider)(implicit executionContext: ExecutionContext) extends AbstractPostgresKgStore(dbConfigProvider) with KgCommandStore {
+  import profile.api._
+
+  private final class PostgresKgCommandStoreTransaction extends KgCommandStoreTransaction {
+    private final implicit class KgEdgeWrapper(edge: KgEdge) {
+      def toRow: KgEdgeRow = KgEdgeRow(
+        id = edge.id,
+        objectKgNodeId = edge.`object`,
+        predicate = edge.predicate,
+        sentences = edge.sentences.mkString(SentencesDelimString),
+        subjectKgNodeId = edge.subject
+      )
+    }
+
+    private final implicit class KgNodeWrapper(node: KgNode) {
+      def toRow: KgNodeRow = KgNodeRow(
+        id = node.id,
+        inDegree = node.inDegree.map(_.toShort),
+        outDegree = node.outDegree.map(_.toShort),
+        pageRank = node.pageRank.map(_.toFloat),
+        pos = node.pos,
+        wordNetSenseNumber = node.wordNetSenseNumber.map(_.toShort)
+      )
+    }
+
+    override def clear(): Unit = {
+//      TODO
+    }
+
+    final private def generateEdgeInsert(edge: KgEdge) =
+      List(kgEdges += edge.toRow) ++
```

Comment:
The mix of kg* prefixes and names without is jarring. That's why I suggested doing without. The Kg* prefixes are really just so other apps can reuse this code without duplicate class names (QueryStore, CommandStore, ... -> KgQueryStore, KgCommandStore).

Reply:
Yeah that makes sense, removed